README.md

```diff
@@ -39,71 +38,36 @@ To deploy your test code within the cluster, you must first build and push a con
 Once pushed, override the image within the deployment to deploy your code for testing:
 
 ```console
-$ oc set image -n machine-api-operator deployment/cluster-autoscaler-operator cluster-autoscaler-operator=<YOUR CONTAINER IMAGE>
-deployment.apps/cluster-autoscaler-operator image updated
+oc set image -n machine-api-operator deployment/cluster-autoscaler-operator cluster-autoscaler-operator=<YOUR CONTAINER IMAGE>
 ```
 
 ### Deploy the code to test locally
 
 To deploy your test code locally, you must first disable the existing operator running within the OpenShift cluster:
 
 ```console
-$ oc scale --replicas 0 -n openshift-machine-api deployments/cluster-autoscaler-operator
-deployment.apps/cluster-autoscaler-operator scaled
+oc scale --replicas 0 -n openshift-machine-api deployments/cluster-autoscaler-operator
 ```
 
 Once the operator has been disabled, build your code to test locally and run it on your machine,
 pointing it to the cluster by passing the appropriate `--kubeconfig` flag:
 
 ```console
-$ make build
-  ...
-$ ./bin/cluster-autoscaler-operator --kubeconfig=<PATH/TO/YOUR/CLUSTERS/KUBECONFIG> ...
+make build
+...
+./bin/cluster-autoscaler-operator --kubeconfig=<PATH/TO/YOUR/CLUSTERS/KUBECONFIG> ...
 ```
 
 ### Build the e2e tests
 
 ```console
-$ make build-e2e 
-go test -c -o bin/e2e github.com/openshift/cluster-api-actuator-pkg/pkg/
+make build-e2e 
 ```
 
 ### Run the autoscaler e2e tests
 
 ```console
-$ NAMESPACE=kube-system ./hack/ci-integration.sh -focus "Autoscaler should" -v -dryRun
-Running Suite: Machine Suite
-============================
-Random Seed: 1617813491
-Will run 4 of 33 specs
-
-SSSSSSSSSSSSSSSSSS
-------------------------------
-[Feature:Machines] Autoscaler should use a ClusterAutoscaler that has 100 maximum total nodes count 
-  It scales from/to zero
-  /cluster-api-actuator-pkg/pkg/autoscaler/autoscaler.go:306
-•
-------------------------------
-[Feature:Machines] Autoscaler should use a ClusterAutoscaler that has 100 maximum total nodes count 
-  cleanup deletion information after scale down [Slow]
-  /cluster-api-actuator-pkg/pkg/autoscaler/autoscaler.go:370
-•
-------------------------------
-[Feature:Machines] Autoscaler should use a ClusterAutoscaler that has 12 maximum total nodes count and balance similar nodes enabled 
-  scales up and down while respecting MaxNodesTotal [Slow][Serial]
-  /cluster-api-actuator-pkg/pkg/autoscaler/autoscaler.go:521
-•
-------------------------------
-[Feature:Machines] Autoscaler should use a ClusterAutoscaler that has 12 maximum total nodes count and balance similar nodes enabled 
-  places nodes evenly across node groups [Slow]
-  /cluster-api-actuator-pkg/pkg/autoscaler/autoscaler.go:605
-•SSSSSSSSSSS
-Ran 4 of 33 Specs in 0.000 seconds
-SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 29 Skipped
-PASS
-
-Ginkgo ran 1 suite in 1.887727166s
-Test Suite Passed
+NAMESPACE=kube-system ./hack/ci-integration.sh -focus "Autoscaler should" -v --dry-run
```

Comment:
I would keep the `Running Suite: ...` and all the lines below, as they give an example of the output the user can expect from a test run.
Maybe we could put them in a separate code section and mention they are an example output rather than having them in the same block where the command is, to not trick the user. WDYT?

Reply:
> Maybe we could put them in a separate code section and mention they are an example output rather than having them in the same block where the command is, to not trick the user. WDYT?

+1 from me

Reply:
Thanks, I agree with you. I've added it back in now in it's own section